Source/ProgressionSystemRuntime/Public/Data/PSWorldSubsystem.h

```diff
@@ -144,6 +153,17 @@ class PROGRESSIONSYSTEMRUNTIME_API UPSWorldSubsystem : public UWorldSubsystem
 	/** Clears all transient data created by this subsystem. */
 	virtual void Deinitialize() override;
 
+	/** Invoked after a game feature plugin is unloaded */
+	virtual void OnGameFeatureUnloading(const UGameFeatureData* GameFeatureData, const FString& PluginURL) override;
+
+	/** Invoked in the early stages of the game feature plugin loading phase */
+	virtual void OnGameFeatureLoading(const UGameFeatureData* GameFeatureData, const FString& PluginURL) override;
+
+	/** Is called to initialize the world subsystem */
+	UFUNCTION(BlueprintCallable, Category= "C++", meta = (BlueprintProtected))
+	void WolrdSubSystemInitialize();
```

Comment:
- `World`, not `Wolrd`
- This function is more reactive, event-like. Please add `On` prefix and make as `BlueprintNativeEvent`, so BP developers will have proper place where to put their logic on initializing the PS module
- Please add more informative comment that it's best place to handle beginplay-like logic form PS module blabla

Reply:
done: https://github.com/h4rdmol/ProgressionSystem/pull/26/commits/ff6209ed44e23aaf0c8dfa5f29b38b90fd93812e https://github.com/h4rdmol/ProgressionSystem/pull/26/commits/98a52a52e46d8b545145877b31051ef8a911efb1